.gitignore

```diff
@@ -1,3 +1,5 @@
 # Need a config/environment.rb for rails.vim to work in engines
 config/environment.rb
 *gem
+.idea
+.bundle
```

Comment:
I'm not that bothered, it's a nitpick, but it feels like commit churn. If it was the only thing I'd pass, but since you're going to be squashing the other things... ;)